packages/admin/cms-admin/src/dam/FileForm/EditFile.tsx

```diff
@@ -195,7 +195,6 @@ const EditFileInner = ({ file, id }: EditFileInnerProps) => {
                 // override default onAfterSubmit because default is stackApi.goBack()
                 // https://github.com/vivid-planet/comet/blob/master/packages/admin/src/FinalForm.tsx#L53
             }}
-            validateOnBlur
```

Comment:
Why did we have this in the first place? 🤔

Reply:
This was a previous attempt to workaround the required issue (but it only made it 50% less shitty): https://github.com/vivid-planet/comet/pull/1341#discussion_r1369638213

With the custom required validator, we can remove that again